tests/functions/test_groupby_agg.py

```diff
@@ -59,3 +59,33 @@ def test_groupby_agg_multi():
     expected_agg = np.array([1, 1, 1, 1, 1, 1])
 
     np.testing.assert_equal(df_new["date_average"], expected_agg)
+
+
+@pytest.mark.functions
+def test_groupby_agg_multi_column():
+    """Test for the case when we want to groupby one column and agg on another, while leaving out other columns."""
```

Comment:
Final nit - I couldn't check how long this line was in the GitHub web interface. This line is the only one failing atm - because it's >79 characters. Can you split it into two lines, run make check, and fix any errors that show up before pushing?